product-ordering-management/src/test/java/org/fiware/tmforum/productordering/ProductOrderingApiIT.java

```diff
@@ -235,6 +235,30 @@ private static Stream<Arguments> provideValidProducts() {
 								.productOrderItem(List.of(productOrderItemVO))
 								.billingAccount(null)));
 
+		ProductOrderItemVO productOrderItemOfferingVO = ProductOrderItemVOTestExample.build().atSchemaLocation(null)
+				.action(OrderItemActionTypeVO.ADD)
+				.id("urn:order-item")
+				.appointment(null)
+				.billingAccount(null)
+				.product(null)
+				.productOffering(ProductOfferingRefValueVOTestExample.build().id("urn:product-offering"))
+				.productOfferingQualificationItem(null)
+				.quoteItem(null);
+
+	/**
```

Comment:
You have direct access to the broker, therefor can create anything in the context you want;) See for example https://github.com/FIWARE/tmforum-api/blob/main/customer-bill-management/src/test/java/org/fiware/tmforum/customerbillmanagement/CustomerBillApiIT.java#L80